website_v2/gcloud-powershell-website-v2/www/static/app/mainController.js

```diff
@@ -0,0 +1,19 @@
+var app = angular.module('powershellSite');
+
+app.controller('MainController',
+    function($scope, $route, $routeParams, $location, $http) {
+    $scope.$route = $route;
```

Comment:
I'm okay leaving 'productInfo' on the scope but why do we need route, location, routeParams on the scope? Where are these used?